examples/kitchensink-ts/package.json

```diff
@@ -22,7 +22,7 @@
     "uuid": "getstarted"
   },
   "engines": {
-    "node": ">=14.19.1 <=18.x.x",
+    "node": ">=14.19.1 <=20.x.x",
```

Comment:
Should we update the minimum here as well? 🤔

Reply:
Oh that's was a pending question in your PR description ! My bad. 
I think we should update min engine if we really want to drop Node14 support.

Reply:
```suggestion
    "node": ">=16.x.x <=20.x.x",
```